axelrod/tests/strategies/test_meta.py

```diff
@@ -585,3 +585,77 @@ def test_strategy(self):
         actions = [(C, C), (C, D), (D, C), (D, D), (D, C)]
         self.versus_test(opponent=axelrod.Alternator(),
                          expected_actions=actions)
+
+"""Tests for the Memory Decay strategy"""
+
+class TestMemoryDecay(TestPlayer):
+
+    name = "Memory Decay: 0.1, 0.03, -2, 1, <class 'axelrod.strategies.titfortat.TitForTat'>, 15"
+    player = axelrod.MemoryDecay
+    expected_classifier = {
+        'memory_depth': float('inf'),
+        'long_run_time': False,
+        'stochastic': False,
```

Comment:
This will be need to be changed to `True`

Reply:
I've changed it to `True` in both `test_meta.py` and `meta.py`, and I get this error:

```
======================================================================
FAIL: test_initialisation (axelrod.tests.strategies.test_meta.TestMemoryDecay)
Test that the player initiates correctly.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/denis/Documents/Axelrod/axelrod/tests/strategies/test_player.py", line 372, in test_initialisation
    self.classifier_test(self.expected_class_classifier)
  File "/home/denis/Documents/Axelrod/axelrod/tests/strategies/test_player.py", line 512, in classifier_test
    self.assertEqual(expected_class_classifier, self.player.classifier)
AssertionError: {'mem[50 chars]ic': False, 'makes_use_of': set(), 'inspects_s[66 chars]alse} != {'mem[50 chars]ic': True, 'makes_use_of': set(), 'inspects_so[65 chars]alse}
  {'inspects_source': False,
   'long_run_time': False,
   'makes_use_of': set(),
   'manipulates_source': False,
   'manipulates_state': False,
   'memory_depth': inf,
-  'stochastic': False}
?                ^^^^

+  'stochastic': True}
?                ^^^


----------------------------------------------------------------------
Ran 2771 tests in 441.809s

FAILED (failures=1, expected failures=1)
```

Reply:
Ah, yes I think I know why that is (the `init` of the parent class is overwriting it).

Add (to the init):

```
572         super().__init__(team = [start_strategy])
573         self.classifier["stochastic"] = True
```

Reply:
I'll try it now. Another thing I've noticed is that I used the `TestPlayer` as the parent class, not the `TestMetaPlayer`. Could that be the cause of the issue?

Reply:
Your solution solves the error. Should I stick with it or try changing the parent class to `TestMetaPlayer`? Or both?

Reply:
Let's stick with `TestPlayer`, this player is a bit different so I think it makes sense :)

Reply:
Okay. I've added the code suggested for `player.py` and am currently running the tests on my machine. I'll report if there are any errors.